### DISCUSSION TITLE
What has been updated recently?

### DISCUSSION THREAD

**Discussion by Pathsis** [linux-surface/linux-surface#1431] [General]
I noticed that the kernel has been silently updated to 6.8.8, but there are no instructions. 

Thanks for the team's hard work, but perhaps some update notes would let us know more details about the results of your work and what improvements our devices might have gotten running Linux.

**Comment by a-andreyev** [comment]
Hi! I'm not a maintainer, just a linux surface user.

I guess the current solution is to look at the changes list via commits history. For example, here's the github view:
https://github.com/linux-surface/linux-surface/commits/master/

As I could see, there're few commits between `6.8.6` and `6.8.8` version. And here're the notes from the commit messages: 

- Build-in basic pinctrl/serial modules to prevent race-conditions in drivers relying on them (like soc-button-array, SAM). See also: https://github.com/linux-surface/linux-surface/pull/1426
- Update v6.8 patches (kernel: https://github.com/linux-surface/kernel/commit/63781a056a0f06d5cc38d7755e0753c7ec08c51c):
  - Fixed IPTS not compiling with -Werror enabled (by jasisonee)
  - Added QuickSPI support to ITHC for Surface Laptop Studio 2 (by quo)

**Comment by StollD** [comment]
Most of the updates are just the regular linux-stable updates. When there are important changes we post them in https://github.com/linux-surface/linux-surface/issues/96.

**Reply by Pathsis** [reply]
Thanks for the reply! I see #96  hasn't been updated in a while either though.
